ui/sitemaps.md

```diff
@@ -618,11 +628,18 @@ The icon may be tinted depending on the state as well.
 **Label and Value Color Syntax:**
 
 ```java
-labelcolor=[item_name operator value = "color", ... ]
-valuecolor=[item_name operator value = "color", ... ]
-iconcolor=[item_name operator value = "color", ... ]
+labelcolor=[item_name operator value = "color", item_name operator value AND item_name operator value = "color", ... ]
+valuecolor=[item_name operator value = "color", item_name operator value AND item_name operator value = "color", ... ]
+iconcolor=[item_name operator value = "color", item_name operator value AND item_name operator value = "color",... ]
 ```
 
+You set as many conditions as you want, along with a color for each condition.
+
+A condition can be a single comparison or a combination of several comparisons separated by the AND operator.
+You can use as many AND-separated comparisons as you want.
```

Comment:
same as above

Reply:
I removed this sentence, I believe it is already clear with the previous sentence.